### PR TITLE
SOP: Branching Fedora IoT

### PR DESCRIPTION
**Pull request by paulwhalen**
Add a new section for working group standard operating procedures, and our first SOP for branching.

### CODE REVIEW

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc [hunk 1]
```diff
@@ -0,0 +1,189 @@
+= Branching Fedora IoT
+
+Branched is the name given to a version of Fedora that has "branched" from the rolling Rawhide tree and will become the next stable Fedora release. This document will detail the steps for branching Fedora IoT from Rawhide. 
+
+To complete these steps and open a pull request you should only need a valid [Fedora account](https://docs.fedoraproject.org/en-US/fedora-accounts/user/).
```
Comment (nullr0ute) [inline user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:5]: Should likely read "and open any pull requests you need" as there's more than one needed.

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+This is not done as a PR and should be pushed directly to the upstream repository. 
+
+----
+git checkout main; git pull; git checkout -b f$release; git push --set-upstream upstream f$release
+----
+
+Example, used for Fedora 40:
+
+----
+git checkout main; git pull; git checkout -b f40 ; git push --set-upstream upstream f40
```
Comment (nullr0ute) [inline user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:26]: It should be noted that creating a new branch on origin currently can not be done by a PR so repo access is needed here.

**Review by nullr0ute — COMMENTED** [review]
A couple of minor bits.

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+----
+
+=== Rawhide (main)
+
+On your local fork, create a new branch for changes to Rawhide (main).
+
+Update the signing key for Rawhide from the main fedora-pungi repo - https://pagure.io/pungi-fedora/commits/main.
+As of Fedora 40 branching you can find this [here](https://pagure.io/pungi-fedora/blob/main/f/fedora.conf=_21).
```
Comment (miabbott) [inline user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:34]: Looks like the URL got munged and dropped the `#`

Correct URL - https://pagure.io/pungi-fedora/blob/main/f/fedora.conf#_21

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+sed -i 's|/pub/alt/iot/rawhide/|/pub/alt/iot/branched/|g' twoweek-nightly.sh
+----
+
+Commit the changes:
+
+----
+git commit -a -m "F-40: Update for branching" -s
+----
+
+Review the changes and if satisfied, push to your fork and open a pull request for others to review.
```
Comment (miabbott) [inline user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:123]: Maybe link to an example PR?

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+Write the commit message:
+
+----
+git commit -a -m "Setup for F-40 branched" -s
+----
+
+Review the changes and if satisfied, push to your fork and open a pull request for others to review.
+
+== Additional Checks
+* check to make sure the Fedora IoT tag has been created in koji
```
Comment (miabbott) [inline user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:183]: Where do I do this?  How can I confirm it was created?

**Review by miabbott — CHANGES_REQUESTED** [review]
Thanks for putting this in the open!  Just a couple of comments from me.

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc (hunk 5 above)
Comment (nullr0ute) [reply user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:183]: Fedora rel-eng does it as part of their SOP so we don't need to, we can check it in koji  (either webux or cli).

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc (hunk 1 above)
Comment (paulwhalen) [reply user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:5]: Added.

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc (hunk 2 above)
Comment (paulwhalen) [reply user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:26]: Mentioned above this, added a note about permissions.

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc (hunk 3 above)
Comment (paulwhalen) [reply user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:34]: thanks, that was my sed command

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc (hunk 4 above)
Comment (paulwhalen) [reply user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:123]: Added

File: user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc (hunk 5 above)
Comment (paulwhalen) [reply user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc:183]: > Where do I do this? How can I confirm it was created?

Clarified the package needed and updated the command.

**Comment by nullr0ute** [comment]
LGTM!

**Pull request merged**